index.js

```diff
@@ -18,8 +18,7 @@ function equal(a, b) {
     if (Array.isArray(a)) {
       length = a.length;
       if (length != b.length) return false;
-      for (i = length; i-- !== 0;)
-        if (!equal(a[i], b[i])) return false;
+      for (i = length; i-- !== 0; ) if (!equal(a[i], b[i])) return false;
```

Comment:
Sorry, it looks like my IDE prettier settings did a bit of reformatting here. If there's a particular code style that's meant to be followed, would be good to get a prettier config into the repo

Reply:
As the source code comments indicate, we track upstream source code from https://github.com/epoberezkin/fast-deep-equal and then place our updates inside. Community PRs usually just attempt to update our specific modifications here or update the upstream library first.